relay-event-normalization/src/event.rs

```diff
@@ -818,6 +818,16 @@ pub fn normalize_performance_score(
                     );
                 }
                 if should_add_total {
+                    if profile.version.is_some() {
+                        event
+                            .tags
```

Comment:
Do you actually need this as a tag on the event payload in the product? Or would a context or an entry in `event.data` be sufficient?

Reply:
I thought about the pros and cons and while I would prefer the tag not to appear to the user in the UI, we still want it to be usable as a filter which means that event context or `event.data` aren't viable options, correct me if I'm wrong.

I can see a use-case where we want to filter the sampled events to get only ones for a specific score version since it doesn't get recalculated on new profiles.

Are there any other large cons to consider when adding this as a tag? If this is something we can easily change in the future, then I'm okay with keeping it as a tag if you agree.

Reply:
> I would prefer the tag not to appear to the user in the UI, we still want it to be usable as a filter

I think using tags for this purpose is discouraged, but I cannot find any docs on it right now. Could you please consult with the Search team on this? AFAIK contexts are indexed.

Reply:
contexts are not indexed by default, we instead promote certain contexts fields to be indexed/searchable or turn them into tags. For example with browser name/version from browser context.

if the product requires it to be searchable, then having it be a context that is promoted to be a tag makes most sense to me.

Reply:
We have https://www.notion.so/sentry/A-guide-on-how-to-make-something-searchable-in-Discover-2f74c50fbbbe47988e723c59733307b8 - but not great docs outside of that. I think we need to prioritize getting something written down for this.